images/linux/scripts/installers/runner-package.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash -e
+################################################################################
+##  File:  runner-package.sh
+##  Desc:  Downloads and Installs runner package
+################################################################################
+
+# Source the helpers for use with the script
+source $HELPER_SCRIPTS/install.sh
+
+DOWNLOAD_URL=$(get_github_package_download_url "actions/runner" "test(\"actions-runner-linux-x64-[0-9].[0-9]{3}.[0-9].tar.gz\")" "latest")
```

Comment:
```suggestion
DOWNLOAD_URL=$(get_github_package_download_url "actions/runner" 'test("actions-runner-linux-x64-[0-9]+\\.[0-9]{3}\\.[0-9]+\\.tar\\.gz")')
```

Reply:
Hey @pshepherd10! Should help with the error. The syntax of JQ, bash and pipe with special characters make things painful to troubleshoot here. Also possible to replace it with the release tag:
```bash
TAG=$(curl -fsSL "https://api.github.com/repos/actions/runner/releases/latest" | jq -r ".tag_name" | tr -cd '0-9.')
DOWNLOAD_URL="https://github.com/actions/runner/releases/latest/download/actions-runner-linux-x64-${TAG}.tar.gz"
```